<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QRadioButton" name="encryptRadioBtn">
            <property name="text">
             <string>Зашифровать</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="decryptRadioBtn">
            <property name="text">
             <string>Расшифровать</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QComboBox" name="modeComboBox">
        <item>
         <property name="text">
          <string>ECB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CBC</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CFB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OFB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CTR</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="loadFileBtn">
          <property name="text">
           <string>Загрузить из файла</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveFileBtn">
          <property name="text">
           <string>Сохранить в файл</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="inputLabel">
          <property name="text">
           <string>Исходный текст</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="inputText"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="keyLabel">
          <property name="text">
           <string>Ключ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="keyText"/>
        </item>
        <item>
         <widget class="QPushButton" name="formKeyBtn">
          <property name="text">
           <string>Сформировать ключ</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="resultLabel">
          <property name="text">
           <string>Результат</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="resultText"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="showResultBtn">
      <property name="text">
       <string>Выполнить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
